nix/haskell-pins.nix

```diff
@@ -93,9 +93,9 @@ let
     };
     amazonka = {
       src = fetchgit {
-        url = "https://github.com/wireapp/amazonka";
```

Comment:
Once the PR is merged, is it safe to delete our fork of Amazonka?

Reply:
Maybe let's not delete it, because that would break older revisions of wire-server code. But we can archive it to suggest that we don't use it anymore.